hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestDFSUtil.java

```diff
@@ -1137,4 +1139,24 @@ public void testAddTransferRateMetricForInvalidValue() {
     DFSUtil.addTransferRateMetric(mockMetrics, 100, 0);
     verify(mockMetrics, times(0)).addReadTransferRate(anyLong());
   }
+
+  @Test
+  public void testGetHostSet() {
+    String[] testAddrs = new String[] {NS1_NN_ADDR, NS1_NN1_ADDR};
```

Comment:
this test case is a bit confusing. Can we just use "unreachable-host1.com:9000" instead?